memberships/views.py

```diff
@@ -141,3 +144,33 @@ def stripe_webhook(request):
         return stripe_webhook.handle(event)
     except ValueError as e:
         return HttpResponse("Failed to parse stripe payload", status=400)
+
+
+@login_required()
+def details_view(request):
+    return render(request, "memberships/member_details.html", {
+        "form": MemberDetailsForm(instance=request.user.member),
+        "profile_image": request.user.member.profile_image
+    }
+                  )
```

Comment:
Just a small formatting thing

```suggestion
    return render(request, "memberships/member_details.html", {
        "form": MemberDetailsForm(instance=request.user.member),
        "profile_image": request.user.member.profile_image
    })
```